### DISCUSSION TITLE
Updating Nodes SO 2.3.51

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#4302] [Unsupported Versions] [answered]
When a Fwd Node comes back online after updating the rest of my Grid will it update automatically from 2.3.21 to 2.3.51 or will I need to perform extra steps to update it? The Fwd node was offline during upgrade.....

--Update--

After rebooting I get the Fault error in grid for my second FWD node. Any thoughts??

![image](https://user-images.githubusercontent.com/70167373/119684627-a745f280-be12-11eb-8754-ee51d66981e4.png)

--Update--

Everything is all good now. Took a few mins for Fault error to clear.

![image](https://user-images.githubusercontent.com/70167373/119685018-fee45e00-be12-11eb-81bc-ac987b520056.png)

**Comment by dougburks** [comment, marked as answer]
Marking this as answered.
